.idea/gradle.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I'm on Android Studio 2.3 beta 3 (Preview), It keeps adding that for some reason.